bench/version-tests/benchmarks.js

```diff
@@ -1,33 +1,34 @@
 // @flow
 
-import mapboxgl from '../src';
-import accessToken from './lib/access_token';
+import mapboxgl from '../../src';
+import { accessToken, styleURL } from '../lib/parameters';
 mapboxgl.accessToken = accessToken;
 
 window.mapboxglVersions = window.mapboxglVersions || [];
 window.mapboxglBenchmarks = window.mapboxglBenchmarks || {};
 
+const url = styleURL();
 const version = process.env.BENCHMARK_VERSION;
 window.mapboxglVersions.push(version);
 
 function register(Benchmark) {
```

Comment:
Feel free to change the parameter type from a class to an instance, so that different subclasses can accept different constructor parameters. And keep pulling out things that differ between version and style benchmarks.

```
const style = 'mapbox://styles/mapbox/streets-v10';
const center = [-77.032194, 38.912753];
const locations = [4, 8, 11, 13, 15, 17].map(zoom => ({zoom, center, style});
register(new Layout(url));
register(new Paint(locations));
register(new QueryPoint(locations));
```